fern/ir-types-v26/definition/http.yml

```diff
@@ -145,6 +145,14 @@ types:
     extends: commons.WithDocs
     properties:
       responseBodyType: types.TypeReference
+      responseProperty:
+        docs: |
+          If set, the SDK will return this property from
+          the response, rather than the response itself.
+
+          This is particularly useful for JSON API structures
+          (e.g. configure 'data' to return 'response.data').
+        type: optional<commons.Name>
```

Comment:
@amckinney i dont want to make you redo a bunch of work -- but i think the right way to represent this in the IR is a union 

```
JsonResponse: 
  union: 
    response: 
    propertyResponse
```
The visitor pattern would force the generators to visit but adding an optional property would certainly be missed by a generator author on upgrade

Reply:
This would mean IRV27, a reverse migration, and updating the generateIntermediateRepresentation code